### DISCUSSION TITLE
Better Server Mod Rejection

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#131] [Tweak Suggestions]
Perhaps with integration to something like [Pack Crash Info](https://www.curseforge.com/minecraft/mc-mods/pack-crash-info), be able to ship a modpack with configurable versions on client and server side, so when there's a mod mismatch it will also explain the modpack version mismatch between the client and server to avoid player confusion.

![image](https://user-images.githubusercontent.com/8945347/230733219-73fc5ce7-a47b-4200-af02-386bd1b99478.png)

**Comment by xJon** [comment]
Another relevant example, player could not tell which version the server was and kept getting Server Mod rejections:
![image](https://user-images.githubusercontent.com/8945347/230799315-ca0b8d3b-96fd-4561-b336-e39e6addc828.png)

**Comment by IntegerLimit** [comment]
This does seem like a good idea.

**Comment by xJon** [comment]
[Better Compatibility Checker](https://legacy.curseforge.com/minecraft/mc-mods/better-compatibility-checker) seems to offer a similar feature, but is only 1.16.5+
